cjss.js

```diff
@@ -1,44 +1,76 @@
-document.addEventListener('DOMContentLoaded', function (event) {
```

Comment:
If you remove the wait until `DOMContentLoaded`, this won't scan any stylesheets added after this script is loaded (unless the script is loaded as `<script … defer>`